app/assets/stylesheets/fuel/base/_typography.scss

```diff
@@ -0,0 +1,102 @@
+// Base Typography
+// ========================================
+
+html {
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+body {
+  @include font-type($primary-sans-serif, normal, 400);
+  @include s-foxtrot;
+  overflow-x: hidden;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  @include font-type($primary-sans-serif, normal, 700);
+  @include rem(margin-bottom, 18px);
+  color: $black-base;
+  line-height: 1.0;
```

Comment:
Properties should be ordered color, -webkit-font-smoothing, line-height, text-rendering<br>`1.0` should be written without the mantissa as `1`